app/src/main/res/values-fr/strings.xml

```diff
@@ -90,14 +91,32 @@
     <string name="manage_staff_screen_guest_list_name">Liste des invités:</string>
     <string name="manage_staff_screen_empty_guest_list">Il n\'y a pas d\'invités à cette événement</string>
 
-    <!--  Supplies  -->
-    <string name="event_creation_screen_supplies">Courses</string>
-    <string name="event_creation_screen_add_supplies">Ajouter des courses</string>
-    <string name="event_details_screen_supplies_button">Trucs à amener</string>
+    <!-- Chimpagne -->
+    <string name="chimpagne_cancel">Annuler</string>
+    <string name="chimpagne_edit">Éditer</string>
+    <string name="chimpagne_save">Sauvegarder</string>
+    <string name="chimpagne_confirm">Confirmer</string>
+    <string name="chimpagne_add">Ajouter</string>
+    <string name="chimpagne_delete">Supprimer</string>
+    <string name="chimpagne_you">Vous</string>
+
+    <!--    Supplies    -->

```

Comment:
Comments in resource strings are a very welcome addition.
Indeed, these files were getting too disorganized.